contrib/platform/src/com/sun/jna/platform/unix/X11.java

```diff
@@ -797,6 +797,8 @@ void XSetWMProperties(Display display, Window window, String window_name,
                           Pointer class_hints);
 
     int XSetWMProtocols(Display display, Window window, Atom[] atom, int count);
+    int XGetWMProtocols(Display display, Window w, Atom[] protocols_return, IntByReference count_return);
```

Comment:
You can see a pattern in the com.sun.platform.win32 package. In general there is the library binding (Kernel32.java) and there is the abstract helper class (Kernel32Util.java) that has "Util" as suffix and contains helpers as static methods.